samples/build.gradle

```diff
@@ -0,0 +1,7 @@
+apply from: '../kotlin.gradle'
+
+dependencies {
+	implementation project(':types')
+    implementation "junit:junit:4.13"
```

Comment:
testImplementation? if so, then it's already in kotlin.gradle

Reply:
Done